docroot/modules/custom/prisoner_hub_taxonomy_sorting/src/Controller/PageController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Drupal\prisoner_hub_taxonomy_sorting\Controller;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\taxonomy\TermInterface;
+use Drupal\views\Views;
+
+/**
+ * Provides route responses for the Example module.
```

Comment:
guess this is a bit out of date

Reply:
That's me copy pasting code.  I'll update it.